CRM/Event/Form/Registration/Register.php

```diff
@@ -467,25 +467,24 @@ public function buildQuickForm() {
 
       // CRM-11182 - Optional confirmation screen
       // Change button label depending on whether the next action is confirm or register
+      $buttonParams = [
+        'type' => 'upload',
+        'spacing' => '&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;',
```

Comment:
I gotta say, I really love the string `'&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;'`

Reply:
Yeah I had to restrain myself from trying to fix everything (or even address this while switching out icons), but there is a lot of this stuff all over.